styles.css

```diff
@@ -356,175 +324,118 @@ button.see-prj:disabled {
   background-color: #2a2b33;
 }
 
-/* background shapes */
-
-.eclipse1 {
-  position: absolute;
-  width: 39.6px;
-  height: 482.57px;
-  border-radius: 48%;
-  background: #edecff;
-  filter: blur(20px);
-  transform: rotate(-36.8deg);
-}
-
-.eclipse2 {
-  position: absolute;
-  width: 70.28px;
-  height: 261.6px;
-  left: 730px;
-  top: 2150.23px;
-  border-radius: 50px;
-  background: #edecff;
-  filter: blur(30px);
-  transform: rotate(36.8deg);
-}
-
-.eclipse3 {
-  position: absolute;
-  width: 308px;
-  height: 155px;
-  right: 593px;
-  top: 2990px;
-  border-radius: 50px;
-  background: rgba(237, 236, 255, 0.3);
-  filter: blur(20px);
-}
-
-.eclipse4 {
-  position: absolute;
-  width: 70.28px;
-  height: 261.6px;
-  left: 770px;
-  top: 3500.23px;
-  border-radius: 50px;
-  background: #edecff;
-  filter: blur(30px);
-  transform: rotate(-66.8deg);
-}
-
-/* footer */
+/* ---FOOTER AND FORM AREA--- */
 .footer {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  background-color: #191765;
-  width: 375px;
+  display: grid;
+  grid-template-columns: 1fr;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-bottom: 8px;
 }
 
-/* no use position-absolute */
+.contact-section {
+  display: grid;
+  grid-template-columns: 1fr;
+  margin-bottom: 71px;
+  grid-gap: 40px;
+}
 
-.contact-us-format {
+.contact-instructions {
   display: flex;
   flex-direction: column;
   align-items: center;
-  flex: none;
-  flex-grow: 0;
+  gap: 24px;
+  order: 0;
 }
 
-/* Title fortmat */
 .contact-us {
-  margin-top: 120px;
-  height: 35px;
-  font-family: "Roboto", serif;
+  font-family: 'Roboto', sans-serif;
   font-style: normal;
   font-weight: 500;
   font-size: 32px;
   line-height: 44px;
-  align-items: center;
   text-align: center;
-  color: #9390ff;
-  flex: none;
+  color: #938fff;
+  order: 0;
 }
 
-/* this is the paragraph */
-.offer-help {
-  width: 327px;
-  font-family: "Inter", serif;
+.we-help {
+  font-family: 'Inter', sans-serif;
   font-style: normal;
-  margin-bottom: 40px;
-  font-weight: 500;
+  font-weight: 300;
   font-size: 16px;
-  color: #f5f5ff;
-  right: 0;
-  left: 0;
+  line-height: 24px;
   text-align: center;
-  letter-spacing: 0.37px;
+  color: #f5f5ff;
+  order: 1;
 }
 
-/* this is form frame */
-.contact {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding: 0;
-  gap: 24px;
-  width: 327px;
-  flex: none;
-  flex-grow: 0;
+/* form */
+.form {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 24px;
+  order: 1;
 }
```

Comment:
According to the project requirements, this entire section needs to be aligned using flexbox. Kindly fix this.

<img width="260" alt="Screen Shot 2022-06-07 at 12 25 39" src="https://user-images.githubusercontent.com/82426682/172445203-fd7b9847-4d02-4801-96d3-30471f5ca822.png">